package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tslint-react",
-  "version": "3.3.3",
+  "version": "3.3.4",
```

Comment:
please don't bump the version in this commit. that will be done during the release process.